multires-lyrics-search/Dockerfile

```diff
@@ -1,11 +1,32 @@
-FROM jinaai/jina:0.4.1-devel
+FROM jinaai/jina
 
-ADD requirements.txt .
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
 
-RUN apt-get update && pip install -r requirements.txt
+RUN pip uninstall -y jina && pip install -r requirements.txt && mkdir data && mv lyrics-data.csv data/
 
-COPY . /
+ENV JINA_DATA_PATH='data/lyrics-data.csv'
+ENV JINA_SHARDS='10'
+ENV JINA_MAX_DOCS='10000'
 
-RUN python app.py dryrun
+EXPOSE 65481
 
-ENTRYPOINT ["python", "app.py"]
+RUN python app.py index
+
+ENTRYPOINT ["python", "app.py", "search"]
+
+LABEL author="Jina AI Dev-Team (dev-team@jina.ai)"
+LABEL type="app"
+LABEL kind="example"
+LABEL avatar="None"
+LABEL description="Jina App using recursive Document structure to search for lyrics from a songs database, the database contains 10k songs"
+LABEL documentation="https://github.com/jina-ai/examples/tree/master/multires-lyrics-search"
+LABEL keywords="[examples, faiss, annoy, ANN]"
```

Comment:
minor thing: keywords are wrong

Reply:
ohh, can u provide q suggestion to the keywords so I can merge here. for next time we rebuild it it should be good